package.json

```diff
@@ -48,6 +48,7 @@
   "devDependencies": {
     "@testing-library/jest-dom": "^5.11.1",
     "@testing-library/react": "^11.1.1",
+    "@testing-library/user-event": "^12.2.0",
```

Comment:
I think this is a cool library so we can better express the user interactions. What do you think?

Reply:
That's great! I just learned about it a few days ago. Thanks for adding it!